_posts/0000-01-11-beardofedu.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-![electrocat](https://octodex.github.com/electrocat.png)
+![electrocat](https://octodex.github.com/images/electrocat.png)
```

Comment:
Good find noticing that "images" was missing from the URL.

Reply:
😍 
 👍